sbt-plugin-test/build.sbt

```diff
@@ -66,6 +66,8 @@ lazy val noDOM = project.settings(baseSettings: _*).
         val moduleInitializers = scalaJSModuleInitializers.value
 
         Def.task {
+          import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
The call to `linker.link()` a few lines below (in this `Def.task {}`) should be surrounded with `Await.result()`, otherwise this test is moot.

Reply:
Done.